Libraries/LibWeb/CSS/StyleComputer.cpp

```diff
@@ -1961,9 +1964,10 @@ CSSPixels StyleComputer::relative_size_mapping(RelativeSize relative_size, CSSPi
     VERIFY_NOT_REACHED();
 }
 
-RefPtr<Gfx::FontCascadeList const> StyleComputer::compute_font_for_style_values(StyleValue const& font_family, CSSPixels const& font_size, StyleValue const& font_style, double font_weight, StyleValue const& font_stretch) const
+RefPtr<Gfx::FontCascadeList const> StyleComputer::compute_font_for_style_values(StyleValue const& font_family, CSSPixels const& font_size, StyleValue const& font_style, double font_weight, Percentage const& font_width) const
 {
-    auto width = font_stretch.to_font_width();
+    // FIXME: We round to int here as that is what is expected by our font infrastructure below expects
```

Comment:
Nit: "expected" and "expects" in one sentence. :^)

Looks like that mistake crept in below too.